types/pdfmake/index.d.ts

```diff
@@ -5,289 +5,19 @@
 //                 Enzo Volkmann <https://github.com/evolkmann>
 //                 Andi Pätzold <https://github.com/andipaetzold>
 //                 Neal Mummau <https://github.com/nmummau>
+//                 Jean-Raphaël Matte <https://github.com/jeralm>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.0
 
-/// <reference lib="dom" />
+/// <reference types="pdfkit" />
 
-declare module "pdfmake/build/vfs_fonts" {
-    let pdfMake: {
-        vfs: any;
-        [name: string]: any;
-    };
-}
-
-declare module "pdfmake/build/pdfmake" {
-    let vfs: TFontFamily;
-    let fonts: { [name: string]: TFontFamilyTypes };
-    function createPdf(
-        documentDefinitions: TDocumentDefinitions,
-        tableLayouts?: any,
-        fonts?: any,
-        vfs?: any
-    ): TCreatedPdf;
-
-    type PageSize =
-        "4A0"|"2A0"|"A0"|"A1"|"A2"|"A3"|"A4"|"A5"|"A6"|"A7"|"A8"|"A9"|"A10"|
-        "B0"|"B1"|"B2"|"B3"|"B4"|"B5"|"B6"|"B7"|"B8"|"B9"|"B10"|
-        "C0"|"C1"|"C2"|"C3"|"C4"|"C5"|"C6"|"C7"|"C8"|"C9"|"C10"|
-        "RA1"|"RA2"|"RA3"|"RA4"|
-        "SRA1"|"SRA2"|"SRA3"|"SRA4"|
-        "EXECUTIVE"|"FOLIO"|"LEGAL"|"LETTER"|"TABLOID";
-
-    type PageOrientation = "portrait" | "landscape";
```

Comment:
It looks like you did not add this type to the new interfaces.d.ts.

Reply:
I did here: https://github.com/jeralm/DefinitelyTyped/blob/a542e86b76952f8459e0a23b650358cbd9d6d43c/types/pdfmake/interfaces.d.ts#L61

(Is this the best way to link code? I'm not too experienced with github)